internal/pkg/cli/pipeline_init.go

```diff
@@ -428,7 +333,7 @@ func (o *initPipelineOpts) createPipelineManifest() error {
 
 	var stages []manifest.PipelineStage
 	for _, environmentName := range o.environments {
-		env, err := o.getEnvConfig(environmentName)
+		env, err := o.store.GetEnvironment(o.appName, environmentName)
```

Comment:
Seems like we are getting environments for both `createPipelineManifest` and `artifactBuckets`. Can we get those environments at the beginning of Execute and then pass those `[]config.Environment` as a parameter to those create methods so as to avoid duplication?

Or maybe a cache variable `o.envs: map[string]config.Environment`.

Reply:
I'll do the first one, since we made this change to avoid having o.envs and o.environments. Thanks!

Reply:
Done.